.github/workflows/codeql_reusable.yml

```diff
@@ -1,3 +1,5 @@
+name: CodeQL
```

Comment:
I think the reason I didn't have this here originally was because this text doesn't show up anywhere.  So I omitted it.

Reply:
I'm not 100% certain, but what I could find indicates the name would only replace the path in the GitHub Actions UI once merged to main. (I believe deploy.yml was behaving the same until I gave it a name. Well... in this case it just disappeared, but I suspect as a workflow with no runs it will show up again once it exists on main.)